src/core/Table/Table.test.tsx

```diff
@@ -2373,3 +2373,159 @@ it('should not have `draggable` attribute on columns with `disableReordering` en
   expect(headerCells[3].getAttribute('draggable')).toBe('true'); // Description column
   expect(headerCells[4].getAttribute('draggable')).toBeFalsy(); // View column
 });
+
+it('should render column manager', () => {
```

Comment:
I am not sure whether it is worth having this test when the next one also checks how column manager is renderer.

Reply:
I have this story to make sure the Column Manager renders with the correct menu items that exclude the Selection and Expandable columns. I don't have those columns in the next story so that it's easier to follow

Should I just combine the checks in this story in the `should hide column when selected in column manager` story?